stepfunctions/templates/consignment_export_definition.json.tpl

```diff
@@ -115,6 +115,14 @@
         },
         "TopicArn": "${sns_topic}"
       },
+      "Next": "Export Status Update"
+    },
+    "Export Status Update": {
+      "Type": "Task",
+      "Resource": "arn:aws:lambda:eu-west-2:229554778675:function:tdr-export-status-update-intg",
```

Comment:
The resource arn string should look something like this: `arn:aws:lambda:eu-west-2:{account_id}:tdr-export-status-update-{env}`

The account id and the environment need to be passed into the template as parameters, so it will work for all enviornments.

Reply:
🤦 Ah, sorry about that.